### ISSUE TITLE
TextBuffer.setTextInRange is deprecated.

### ISSUE THREAD

**Issue by viktorsteinwand** [t9md/atom-markdown-toc-auto#15]
The `undo` option is deprecated. Call groupLastChanges() on the TextBuffer afterward instead.
```
TextBuffer.setTextInRange (<embedded>:33539:22)
TextEditor.setTextInBufferRange (<embedded>:3552:35)
insertToc (/Users/Viktor/.atom/packages/markdown-toc-auto/lib/utils.js:143:10)
Object.insertTocAtPoint (/Users/Viktor/.atom/packages/markdown-toc-auto/lib/utils.js:155:3)
HTMLElement.markdown-toc-auto:insert-toc (/Users/Viktor/.atom/packages/markdown-toc-auto/lib/main.js:22:20)
CommandRegistry.handleCommandEvent (<embedded>:23883:49)
```

**Comment by sbkg0002** [comment]
Any update on this?

**Comment by t9md** [comment]
Thank you for pinging me.
I just have released 0.9.1.

**Comment by sbkg0002** [comment]
Thanks for the update!
